index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Middle</title>
+</head>
+<body>
+    <section class="work">
+        <div class="work-title">
+            <h1 class="work-title-h1">My Recent Works</h1>
+            
+        </div>
+        <div class="work-items">
+            <hr>
+        </div>
+        <div class="work-items">
+            <img src="img/ImgPlaceholder.png" alt="">
+            <h2>
+                Multi-Post Stories
+            </h2>
+            <p class="work-items-p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+        </div>
+        <div class="work-items work-items-project">
```

Comment:
- [ ] For every project use `font-size, font-family, font-weight` values same as Figma template.
![Captura de pantalla_2021-09-18_02-28-52](https://user-images.githubusercontent.com/53111464/133880538-2caf0e18-75ee-434d-a110-3550602e08f2.png)

**Figma**
![Captura de pantalla_2021-09-18_02-34-05](https://user-images.githubusercontent.com/53111464/133880570-c9fd27f0-faea-4350-a962-983508f9c513.png)

Reply:
font-family doesn't have

Reply:
I wish you had looked at the css codes and you would have seen the font-family and font-size specification

Reply:
![Captura de pantalla_2021-09-18_12-43-23](https://user-images.githubusercontent.com/53111464/133897737-97a84622-05d1-4449-97c6-72bf5e1e7001.png)

Reply:
Yes I look the style.css file and you are using another `font-family, font size, font wight` you can check the values in Figma in the left sidebar for every template.

Reply:
did you before make with website Figma? Font family inter didn't have and if you are write Inter result missing font 
and if you looked at my CSS you must see that inter on CSS :) pls be careful on the review